docs/UserGuide.md

```diff
@@ -373,7 +373,7 @@ Examples:
 
 #### Adding a visit
 
-Adds a visit by the person, location of visit and date of visit
+When you want to add a visit, you can use the following command to add a visit by the person, location of visit and date of visit
```

Comment:
maybe change to if? since the below part all used if, to make it look more consistent.

Reply:
Fixed.